Framework/API/src/LogManager.cpp

```diff
@@ -337,6 +337,22 @@ LogManager::getTimeSeriesProperty(const std::string &name) const {
   }
 }
 
+/**
+ * Returns the time dependent standard deviation
+ * @param name :: The name of the property
+ * @return A single double value
+ */
+double LogManager::getTimeAveragedStd(const std::string &name) const {
+  auto prop = this->getProperty(name);
```

Comment:
We could simplify this to `return getTimeSeriesProperty(name)->getStatistics().time_standard_deviation` to reduce duplication with getting the `TimeSeriesProperty`?

Reply:
I tried, but I get compile issues:
```
/home/3y9/Mantid/mantid/Framework/API/inc/MantidAPI/LogManager.h:102:3: note:   template argument deduction/substitution failed:
/home/3y9/Mantid/mantid/Framework/API/src/LogManager.cpp:346:43: note:   couldn't deduce template parameter ‘T’
    return this->getTimeSeriesProperty(name)->getStatistics().time_standard_deviation ;
```

Reply:
Ah yes I had forgotten about the template parameter: `return getTimeSeriesProperty<double>(name)->getStatistics().time_standard_deviation` ?

Reply:
It works. The error message is not entirely accurate for time series properties that are not double (it says that are not time series properties at all), but I changed it to your proposed version.